docs/source/pythonapi/examples/post-processing.rst

```diff
@@ -0,0 +1,13 @@
+.. _notebook_post_processing:
+
+===============
+Post Processing
+===============
+
+.. only:: html
+
+    .. notebook:: post-processing.ipynb
```

Comment:
I'm a fan of your new Notebook, but I'm curious why it is the only one included in this section on post processing?